views/partials/compromised_accounts.hbs

```diff
@@ -1,23 +1,36 @@
 <!--if user scanned email from a 'featured Breach' page-->
-{{#if featuredBreach}}
-  {{> featured_compromised_account}}
-{{else}}
-  <h3>Your Compromised Accounts</h3>
-{{/if}}
+
+{{#if foundBreaches}}
+  <div class="compromised-accounts">
+    <div class="section-wrapper">
+      <h3 class="whole section-headline">
+        {{#if userAccountCompromised }}
+          Other Compromised Accounts Founds
```

Comment:
"Founds" sounds exotic. Probably should be singular.

Reply:
I sware I can spel.